modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,46 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-interacting-with-pipelines-using-the-developer-perspective_{context}"]
+= Interacting with Pipelines using the Developer perspective
+
+The *Pipelines* view in the *Developer* perspective lists all the Pipelines in a project along with details, such as the namespace in which the Pipeline was created, the last PipelineRun, the status of the Tasks in the PipelineRun, the status of the PipelineRun, and the time taken for the run.
+
+[Discrete]
+.Procedure
+. In the *Pipelines* view of the *Developer* perspective, select a project from the *Project* drop-down to see the Pipelines in that project.
++
+//image::op-pipeline-list.png[Pipelines in the Developer perspective]
```

Comment:
The image text is small and hard to read, even on a large monitor. I haven't worked with images yet, so I'm not sure what our usage recommendations are. But I can't see the drop-down list at all. I suggest cropping the image to the area you are trying to show.

Reply:
As discussed on slack, I will be replacing the images ( have already commented it out here ).